static/src/javascripts/lib/$.js

```diff
@@ -9,7 +9,7 @@ bonzo.aug({
     },
 });
 
-const $ = (selector: ?string | ?Node, context: ?Element | ?string): bonzo =>
+const $ = (selector: ?string | ?Node, context: ?Node | ?string): bonzo =>
```

Comment:
The typing in this module is inconsistent and, form a cursory inspection, appears overly broad. `Node` can represent a whole bunch of things that you'll never pass to `qwery`. [`HTMLElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) looks more appropriate, but this would need to be enforced consistently throughout the application.

This change makes the code more consistent than it was, so I reckon you can leave it like this. But can you add a comment above this function saying something like:

```js
// #? Is Node the correct type here? HTMLElement may be more appropriate
```

@sndrs @regiskuckaertz @gustavpursche: any thoughts?